scipy/spatial/transform/rotation.py

```diff
@@ -306,6 +351,66 @@ def from_dcm(cls, dcm):
                 <https://en.wikipedia.org/wiki/Rotation_matrix#In_three_dimensions>`_
         .. [2] F. Landis Markley, `Unit Quaternion from Rotation Matrix
                <https://arc.aiaa.org/doi/abs/10.2514/1.31730>`_
+
+        Examples
+        --------
+        >>> import numpy as np
+        >>> from scipy.spatial.transform import Rotation as R
+
+        Initialize a single rotation:
+
+        >>> r = R.from_dcm([
+        ... [0, -1, 0],
+        ... [1, 0, 0],
+        ... [0, 0, 1]])
+        >>> r.as_dcm().shape
+        (3, 3)
+
+        Initialize multiple rotations in a single object:
+
+        >>> r = R.from_dcm([
+        ... [
+        ...     [0, -1, 0],
+        ...     [1, 0, 0],
+        ...     [0, 0, 1],
+        ... ],
+        ... [
+        ...     [1, 0, 0],
+        ...     [0, 0, -1],
+        ...     [0, 1, 0],
+        ... ]])
+        >>> r.as_dcm().shape
+        (2, 3, 3)
+
+        If input matrices are not special orthogonal, then a special orthogonal
```

Comment:
I know what orthogonal is, but "special orthogonal" may deserve an explanation. what is it?